src/pages/profile.tsx

```diff
@@ -0,0 +1,182 @@
+import { GetStaticProps } from 'next';
```

Comment:
Take a screenshot how the site looks like...

Reply:
Okay! Sorry I didn't notice that. I will do it